docs/source/scaleVM.md

```diff
@@ -0,0 +1,165 @@
+# 🖥️ Scale Fedora & Windows VMs with Benchmark Runner
+
+This document provides automated scripts and configuration to **scale Fedora and Windows VMs** on OpenShift clusters. It verifies the VM login page and measures **bootstorm time**—the time it takes for VMs to become fully operational.
+
+---
+
+## 🚀 Features
+
+- Scale Fedora 37 VMs using **ephemeral storage**
+- Scale Windows 11 VMs using **ODF-backed storage** (via DataVolume)
+- Parallel VM loading across specified worker nodes
+- Optional metrics collection via **Elasticsearch**
+- Logs and artifacts are stored locally for post-run analysis
+
+---
+
+## 🧪 Prerequisites
+
+- OpenShift cluster with access to worker nodes
+- [`podman`](https://podman.io) installed
+- Access to the [benchmark-runner](https://quay.io/repository/ebattat/benchmark-runner) container image
+- Properly configured `KUBECONFIG` for cluster access
+
+---
+
+## ⚙️ Parameters
+
+### Common Parameters
+| Variable | Description |
+|----------|-------------|
+| `KUBEADMIN_PASSWORD` | OpenShift admin password |
+| `KUBECONFIG_PATH` | Path to your kubeconfig (e.g. `/root/.kube/config`) |
+| `SCALE_NODES` | List of target worker nodes (e.g. `"[ 'worker-0', 'worker-1' ]"`) |
+| `THREADS_LIMIT` | Number of physical CPUs to use for parallelism |
+| `TIMEOUT` | Maximum duration (in seconds) for the run |
+| `RUN_ARTIFACTS_URL` | (Optional) Remote URL for saving logs |
+| `SAVE_ARTIFACTS_LOCAL` | If true, saves logs locally to `/tmp/benchmark-runner-run-artifacts` |
```

Comment:
It wouldn't be a bad idea for this to optionally be a path to save the logs to.  Although looking further down it's possible to change this by means of volume mapping with podman.